app/src/main/res/layout/fragment_itunes_search.xml

```diff
@@ -16,24 +16,13 @@
 
         <com.google.android.material.appbar.MaterialToolbar
             android:id="@+id/toolbar"
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
```

Comment:
The parent LinearLayout can be removed here

Reply:
fixed